README.md

```diff
@@ -3,6 +3,7 @@ This project deploys the OpenShift Assisted Installer in Minikube and spawns lib
 
 # Prerequisites
 - CentOS 8 or RHEL 8 host
+- Skipper https://github.com/stratoscale/skipper
```

Comment:
skipper will installed in create_full_environment, not sure we need to ask it as Prerequisites. 
If we are adding it here we need to add all the packages we install in create_full_environment